e2e-tests/otel-collector/helm/values.yaml

```diff
@@ -0,0 +1,243 @@
+mode: daemonset
+image:
+  repository: 669783387624.dkr.ecr.us-east-1.amazonaws.com/otel-collector-contrib
+  tag: "otelcolcontrib-v6bc4add4"
+securityContext:
+  runAsUser: 0
+  runAsGroup: 0
+resources:
+  limits:
+    cpu: 512m
+    memory: 2Gi
+useGOMEMLIMIT: true
+presets:
+  logsCollection:
+    enabled: true
+    includeCollectorLogs: true
+    storeCheckpoints: false
+  hostMetrics:
+    enabled: true
+  kubernetesAttributes:
+    enabled: true
+  kubernetesEvents:
+    enabled: true
+  kubeletMetrics:
+    enabled: true
+extraEnvs:
+  - name: POD_IP
+    valueFrom:
+      fieldRef:
+        fieldPath: status.podIP
+  - name: OTEL_RESOURCE_ATTRIBUTES
+    value: "k8s.pod.ip=$(POD_IP)"
+  - name: DD_API_KEY
+    value: "tottooo"
+  - name: OTEL_K8S_NAMESPACE
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: metadata.namespace
+config:
```

Comment:
If possible, it would be great to have this section configurable through collector config yaml files that live in the `tests` directory

Reply:
Added the `WithOTelConfig`flag that you can use the same way as on `datadog-agent`